ffv1.md

```diff
@@ -391,6 +391,10 @@ In JPEG2000-RCT, the coding order would be left to right and then top to bottom,
 
 Y[1,1] Y[2,1] Cb[1,1] Cb[2,1] Cr[1,1] Cr[2,1] Y[1,2] Y[2,2] Cb[1,2] Cb[2,2] Cr[1,2] Cr[2,2]
 
+### Color Filter Array
+
+(TODO)
```

Comment:
what coming here?

Reply:
Some general introduction to CFA, order of planes... as for the 2 other colors spaces.